src/Context/AuthContext.js

```diff
@@ -0,0 +1,29 @@
+import { createContext, useContext } from "react";
+import { useState } from "react";
+
+const AuthContext = createContext();
+
+const AuthContextProvider = ({ children }) => {
+    const getToken = localStorage.getItem("login-Token");
+    const getUserDetails = localStorage.getItem("user");
+
+    const [token, setToken] = useState(getToken || "");
+    const [user, setUser] = useState(getUserDetails || "");
+
```

Comment:
Instead of giving OR you can use optional chaining

Reply:
ok sure